Makefile

```diff
@@ -305,6 +305,7 @@ rpkg:
 	Rscript -e "require(mxnet); mxnet:::mxnet.export(\"R-package\")"
 	Rscript -e "require(roxygen2); roxygen2::roxygenise(\"R-package\")"
 	R CMD build --no-build-vignettes R-package
+	mv mxnet_*.tar.gz mxnet_current_r.tar.gz
```

Comment:
If there's only one wildcard match, this command works. 
I don't think R's native package builder can generate fixed-name tarball.